roles/edpm_update/tasks/packages.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: Apply packages updates
+  ansible.builtin.dnf:  # noqa: package-latest
+    name: "*"
```

Comment:
I am just thinking out loud here. Shouldnt this just update OSP related packages? Because this will unconditionally update everything on the system, which might be undesirable by the end user.

Reply:
Na, definitely need to update RHEL here as part of the minor update process.

Reply:
Quoting the doc:

```
- autoremove
        If `true', removes all "leaf" packages from the system that were originally installed as dependencies of user-installed packages but which are no longer required by any such package.
        Should be used alone or when `state' is `absent'
        default: 'no'
        type: bool
        added in: version 2.4 of ansible-core
```

If we want to use it (and it could have sense, IMHO), we should create another dnf task.